redis/src/parser.rs

```diff
@@ -623,12 +629,15 @@ mod tests {
     #[test]
     fn decode_resp3_big_number() {
         let val = parse_redis_value(b"(3492890328409238509324850943850943825024385\r\n").unwrap();
-        assert_eq!(
-            val,
-            Value::BigNumber(
-                BigInt::parse_bytes(b"3492890328409238509324850943850943825024385", 10).unwrap()
-            )
+        #[cfg(feature = "num-bigint")]
+        let expected = Value::BigNumber(
+            num_bigint::BigInt::parse_bytes(b"3492890328409238509324850943850943825024385", 10)
+                .unwrap(),
         );
+        #[cfg(not(feature = "num-bigint"))]
+        let expected =
+            Value::BigNumber(b"(3492890328409238509324850943850943825024385\r\n".to_vec());
```

Comment:
```suggestion
            Value::BigNumber("3492890328409238509324850943850943825024385".to_string());
```
If num-bigint is going to be optional, I would expect getting the number as a string from the library, instead of getting bytes including CRLF and (.

Reply:
good catch! this revealed the fact that we don't actually test for no default features, and this flow wasn't at all tested!